IFC4x3/Properties/o/OutputCurrent_3hgdbiYpLERhPeEQfTel7i/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OutputCurrent_3hgdbiYpLERhPeEQfTel7i" Name="OutputCurrent" UniqueId="ebaa796c-8b35-4e6e-b668-39aa5da2f1ec" PrimaryDataType="IfcElectricCurrentMeasure" />
```

Comment:
Documentation suggests it should be a bounded property?

Reply:
Agree. Commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d